.artifact/tree.json

```diff
@@ -1,5 +1,5 @@
 {
-  "viam-cartographer": {
+  "slam": {
```

Comment:
I am not exactly sure why this happened, it auto generated after running artifact push and only making changes to the viam-cartographer/mock_lidar directory. Did I miss some clean up that needed to be done?

Reply:
I think that this would only happen if you had lidar readings at `.artifact/data/slam/mock_lidar/`

They should be deleted